dev/pyproject.py

```diff
@@ -134,6 +134,8 @@ def build(package_type: PackageType) -> None:
                     # Required for exporting metrics from the MLflow server to Prometheus
                     # as part of the MLflow server monitoring add-on
                     "prometheus-flask-exporter",
+                    # Required for env management in mlflow.models.predict
+                    "uv<1,>=0.5.5",
```

Comment:
do we need this?

Reply:
If we have some users using `mlflow[extras]`, then they don't need to install uv themselves. Because I think one annoying thing of running pip in databricks is you need to restart python, and all variables are gone.

Reply:
I think we can add this later. We're not sure how many users run `mlflow[extras]`.

Reply:
I'd keep this if uv was a pure python package.

Reply:
Make sense :)

Reply:
We can add it anytime if there is a request.